### DISCUSSION TITLE
New environment variable OFFLINE_MODE explanation.

### DISCUSSION THREAD

**Discussion by gmentzik** [Budibase/budibase#12069] [Help] [answered]
Hi devs.

In docker-compose.yaml there is a new env param called OFFLINE_MODE (OFFLINE_MODE: ${OFFLINE_MODE}) at app and worker service. I don't see a set value for it at .env. Also I cannot find documentation for it. 
Could you please explain what does it do?

**Comment by shogunpurple** [comment, marked as answer]
Hi @gmentzik - it's for enterprise customers who are using an airgapped installation. Safe to ignore if you are on the free/community plan

**Reply by gmentzik** [reply]
Thank you for the prompt reply. But what does it do. Can it make my host more secure by closing some interactions with outside world?

**Reply by shogunpurple** [reply]
No, it's to enable support for offline licensing which is on the enterprise plan

**Reply by gmentzik** [reply]
Ok thank you!
